### ISSUE TITLE
[BITV] 9.4.1.2/3.17 - The elements in the "recommendations" list that are placed under the editor have been implemented as links without the href attribute but with tabindex="0". (2)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/recommendations#594]
However, the elements should be implemented as buttons, not as links.

- [ ] Check implementation https://github.com/nextcloud/recommendations/pull/637

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v3-n17

</details>

- [ ] 9.4.1.2/10.17 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v10-n17

**Comment by Pytal** [comment]
Recomendations not available on master with F2V

**Comment by JuliaKirschenheuter** [comment]
> Recomendations not available on master with F2V

should we close a ticket?

**Comment by Pytal** [comment]
> should we close a ticket?

No, comment is outdated

Was implemented with https://github.com/nextcloud/recommendations/pull/637

**Comment by JuliaKirschenheuter** [comment]
> Was implemented with #637

closing ticket
